### DISCUSSION TITLE
partitioning?

### DISCUSSION THREAD

**Discussion by bobblkabb** [zbm-dev/zfsbootmenu#301] [Questions] [answered]
How do I install the system (encrypted) if I have the following layout:
sdc (SSD) -> EFI/root partition
sda (HD) -> home partition
sdb (HD) -> zpool

**Comment by zdykstra** [comment, marked as answer]
As long as `/` is on ZFS, just put ZBM on your ESP. The number/layout of drives really doesn't matter.
